.setup/bin/sample_courses/models/course/course_create_gradeables.py

```diff
@@ -0,0 +1,415 @@
+
+from __future__ import print_function, division
+from datetime import timedelta
+import hashlib
+import json
+import os
+import random
+import shutil
+import subprocess
+import os.path
+import random
+from tempfile import TemporaryDirectory
+from submitty_utils import dateutils
+
+from sqlalchemy import Table, select
+
+from sample_courses import *
+from sample_courses.utils import mimic_checkout
+from sample_courses.utils.dependent import commit_submission_to_repo
+from sample_courses.utils.create_or_generate import (
+    generate_probability_space,
+    generate_random_ta_comment,
+    generate_versions_to_submit,
+    generate_random_user_id,
+    create_gradeable_submission,
+    create_pdf_annotations
+    )
+
+
+class Course_create_gradeables:
+    """
+    Object that contain functions that adds gradables to the course and database
+    Used as an helper to the Course Class
+    """
+    semester:str
+    # code:dict idk type
+    # instructor:dict idk type
+    gradeables:list
+    make_customization:bool
+    users: list
+    registration_sections: int
+    rotating_sections: int
+    registered_students: int
+    no_registration_sections: int
+    no_rotating_students: int
+    unregistered_students: int
+    def __init__(self):
+        # Anything that needs to be initialized goes here
+        pass
+
+    def add_gradeables(self):
+        anon_ids = {}
+        for gradeable in self.gradeables:
+            #create gradeable specific anonymous ids for users
+            prev_state = random.getstate()
+            for user in self.users:
```

Comment:
In a future PR, I will split this function into parts and move them into more functions and eventually into classes to become more maintainable.